docs/internal/result_handlers.md

```diff
@@ -0,0 +1,307 @@
+```python
+self._client.insert(table_name, result_df)
+```
+
+Command:
+```console
+time data-validation -ll DEBUG validate row -sc=oravol -tc=pg \
+-tbls=dvt_test.tab_vol_1m -pk=id --hash='*' \
+--result-handler=pg_rh.pso_data_validator_results.results \
+2>/tmp/debug.log
+
+real	2m35.404s
+user	1m11.604s
+sys	0m3.503s
+```
+
+DVT stages:
+```
+04/03/2025 11:25:25 AM-DEBUG: Build config elapsed: 0.17s
+04/03/2025 11:25:29 AM-DEBUG: Target query elapsed: 4.55s
+04/03/2025 11:25:32 AM-DEBUG: Source query elapsed: 7.6s
+04/03/2025 11:25:48 AM-DEBUG: Generate report elapsed: 15.8s
+04/03/2025 11:27:58 AM-DEBUG: Write results to PostgreSQL elapsed: 129.41s
+```
+
+`vmstat` output:
+```
+procs -----------memory---------- ---swap-- -----io---- -system-- ------cpu-----
+ r  b   swpd   free   buff  cache   si   so    bi    bo   in   cs us sy id wa st
+ 0  0      0 32094052  21720 558024    0    0    18     7  362  489  8  1 91  0  0
+ 0  0      0 31423908  21728 557608    0    0     0    14 5331 5680 13  3 83  0  0
+ 1  0      0 30860328  21736 557608    0    0     0     6 1450 1929 21  4 75  0  0
+ 1  0      0 30537996  21736 557608    0    0     0     0 1156 1581 24  1 75  0  0
+ 1  0      0 30316580  21744 557608    0    0     0     2 1196 1658 25  0 75  0  0
+ 1  0      0 29349932  21752 557608    0    0     0     3 1206 1687 24  1 75  0  0
+ 1  0      0 28944016  21752 557608    0    0     0     0 1192 1759 10  1 89  0  0
+ 0  0      0 28942720  21760 557608    0    0     0     2 1160 1746  6  0 93  0  0
+ 0  0      0 28942464  21760 557608    0    0     0     0 1092 1611  6  0 93  0  0
+ 0  0      0 28941800  21760 557608    0    0     0    14 1126 1671  6  0 94  0  0
+ 0  0      0 28941800  21760 557608    0    0     0     2 1197 1751  6  0 93  0  0
+ 0  0      0 28941768  21760 557608    0    0     0     0 1112 1590  7  0 93  0  0
+ 1  0      0 28941860  21760 557608    0    0     0     0 1100 1594  6  0 94  0  0
+ 0  0      0 28941796  21760 557608    0    0     0     0 1091 1564  6  0 93  0  0
+ 1  0      0 28941732  21760 557608    0    0     0     0 1087 1548  7  0 93  0  0
+ 0  0      0 28941536  21760 557608    0    0     0     0 1076 1554  6  0 94  0  0
+ 0  0      0 32083492  21760 557608    0    0     0     0 1013 1544  7  0 93  0  0
+ 0  0      0 32083500  21768 557608    0    0     0     1  851 1525  0  0 100  0  0
+```
+
+### PostgreSQL - Dataframe.to_sql(chunksize=1000)
+
+Writing to PostgreSQL using the pandas `to_sql(chunksize=1000)` method.
+```python
+            result_df.to_sql(
+                table_name,
+                self._client.con,
+                schema=schema_name,
+                if_exists="append",
+                index=False,
+                chunksize=1000,
+            )
+```
+
+Command:
+```console
+time data-validation -ll DEBUG validate row -sc=oravol -tc=pg \
+-tbls=dvt_test.tab_vol_1m -pk=id --hash='*' \
+--result-handler=pg_rh.pso_data_validator_results.results \
```

Comment:
```suggestion
--result-handler=<GCP_PROJECT_ID>.pso_data_validator_results.results \
```

Reply:
`rg_rh` is the name of a connection file, not a project name. I've updated the doc.